index.html

```diff
@@ -401,9 +401,9 @@ <h4 class="uppercase mb16">Blogs</h4>
 		{% for blog in site.data.blogs %}
 				{% capture modulo %}{{ forloop.index | modulo: 4 }}{% endcapture %}
 				{% if modulo == '1' %}
-		<div class="row">
+		<div class="row col-xs-12 col-sm-6 col-md-4">
 			{% endif %}
-			<div class="col-md-3">
+			<div>
```

Comment:
Revet this & have a look at https://getbootstrap.com/docs/4.0/layout/grid/ to see how it works :)